website/src/parsers/markdown/transformers/remark/codeExample.txt

```diff
@@ -1,5 +1,8 @@
+const visit = require("unist-util-visit");
```

Comment:
I think we should include a comment here that explains which packages can be required.

Reply:
addressed in 0e12f3c